tests/integration/ha/continuous_writes.py

```diff
@@ -26,6 +27,14 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class ContinuousWritesResult:
+    count: int = -1
+    last_message: Optional[object] = None
+    last_expected_message: int = -1
+    lost_messages: int = -1
```

Comment:
**niptick, minor** I'd rather use Optional instead of -1, or not have any default at all actually